### DISCUSSION TITLE
Experience GitHub Community Discussions IRL at GitHub Universe this week!

### DISCUSSION THREAD

**Discussion by ettaboyle** [community/community#74761] [Other Feature Feedback, Questions, & Ideas]
![AskGitHub2023-Day1_1](https://github.com/community/community/assets/75342807/92ec9605-e87a-4c9e-9ada-1c6b6474012d)

## Ask GitHub: Happening this Week at Universe!

**[GitHub Universe, San Francisco] November 8-9, 2023:** 
Located in the gallery adjacent to the Discussions Lounge + a helpful desk in the Grand Lobby, Ask GitHub has product experts from across the company ready to address your biggest challenges. Stop by – for help with anything from automating workflows, to writing better code with AI, to planning and tracking your work. Our amazing team of subject matter experts are eager to assist and offer tips & best practices to ensure you get the experience of Community Discussions in real life! 

Getting help with product questions, solving business challenges, and learning more about how to be more proactive with your security are top priorities while at GitHub Universe – _and Ask GitHub is the place to do just that_ 

We believe in social coding and troubleshooting with others is the best way to get the most out of GitHub and Universe. Take advantage of working with us in person. 

### Universe 2024 pre-sale tickets are available now. 🎟️ More at the [GitHub Universe](https://reg.githubuniverse.com/flow/github/universe23/attendee-portal/page/welcome) site.
 
<img width="752" alt="Screenshot 2023-11-13 at 12 16 18 PM" src="https://github.com/community/community/assets/75342807/1a0a3c61-4342-4863-bfa7-9d820f2462b3">

![AskGitHub2023-Day1_2](https://github.com/community/community/assets/75342807/256fb460-f72c-4a48-bfa6-926ff1120ad4)

**Comment by ghostinhershell** [comment]
Love to see the IRL engagement amongst such tasteful art 🎨 ❤️

**Comment by samus-aran** [comment]
I love the blue shirts 👕 💙 ❤️

**Comment by lulitzarodgarr** [comment]
🐶
